packages/components/inputs/rich-text-utils/src/rich-text-body/dropdown.styles.tsx

```diff
@@ -2,6 +2,15 @@ import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 import { customProperties as vars } from '@commercetools-uikit/design-system';
 
+type Props = {
```

Comment:
I think it will be nice to rename this to `TDropdownItemProps` or something like that. Depending on how it is used.